dev-cpp/string-theory/string-theory-3.3.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+if [[ ${PV} = 9999 ]]; then
+	  GIT_ECLASS="git-r3"
+	  EXPERIMENTAL="true"
```

Comment:
Seems unused?

Reply:
It was intended for a comming 9999 - I can remove it.